### PR TITLE
flake-module: enable treefmt as default formatter for the flake

### PR DESCRIPTION
**Pull request by Mic92**
Merge https://github.com/numtide/treefmt-nix/pull/28 first

### CODE REVIEW

File: flake-module.nix [hunk 1]
```diff
@@ -15,12 +15,20 @@ in
             Project-level treefmt configuration
           '';
           type = types.submoduleWith {
-            modules = (import ./.).all-modules pkgs;
+            modules = (import ./.).all-modules pkgs ++ [{
+              options.enableAsFlakeFormatter = lib.mkOption {
```
Comment (zimbatm) [inline flake-module.nix:19]:
```suggestion
              options.flakeFormatter = lib.mkOption {
```
Something a bit less verbose like that?

**Review by srid — APPROVED** [review]

**Comment by Mic92** [comment]
Updated the description.

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors merge

**Pull request merged**